A71-xds-fallback.md

```diff
@@ -83,7 +83,9 @@ target.
 The CSDS RPC service will be changed to get data from all XdsClient instances.
 A new string field named `client_scope` will be added to the CSDS
 [ClientConfig] message to indicate the channel target the data is associated
-with.
+with. For gRPC clients, this field will contain the key channels use to lookup
+their XdsClient instance, such as dataplane target for client channels or
+special value for xDS-enabled gRPC servers.
```

Comment:
I think the suggestion from @ejona86 was that we should specify the exact string of the key we are going to use for the servers.

Reply:
We haven't chosen that yet. I suggest we delay this decision in case we see some special requirements, like format or something...

Reply:
I'm not sure what special requirements would arise for this.  Let's just pick a string like "server".

Reply:
I feel like "server" would clash for way too many teams... I propose `grpc::server`.

Reply:
I don't think there's a concern about whatever we choose here conflicting with anything else, since this name is meaningful only in the context of the individual client.  In the case of a gRPC xDS client, we can say that "server" means the gRPC servers, and it wouldn't cause any problems even if other xDS clients used the same string to refer to something else.

"grpc::server" seems too C++-specific.  I think "server" is fine.

Reply:
As per subsequent offline discussion, we need to use a character that would not be valid in an authority, since otherwise there could still be a conflict with a data plane authority (e.g., if the target URI is "xds://xds.example.com/@server").  Let's go with "#server", since "#" is not a valid character in authorities and would therefore be percent-encoded.

Reply:
Done

Reply:
Thanks!

In further offline discussions, we realized that there isn't necessarily a conflict here with the authority characters specifically, since the key for the clients will actually be the full target URI, not just the data plane authority.  But either way, "#server" is a good choice, since it's also not a valid URI.